app/settings.py

```diff
@@ -38,6 +38,7 @@ def get_env_or_fail(key):
     return value
 
 
+CDN_URL = os.getenv("CDN_URL", "https://cdn.ons.gov.uk/sdc")
```

Comment:
I know we discussed this for a while, but I think we need to change this. 

We have two uses of this and we mutate the string in both places - either removing `/sdc` or adding `/design-system`. If we had:

```python
CDN_URL = os.getenv("CDN_URL", "https://cdn.ons.gov.uk")
CDN_ASSETS_PATH = os.getenv("CDN_ASSETS_PATH", "/sdc/design-system")
```

Then it keeps the configuration in one place, and also doesn't enforce a `design-system` folder.

Reply:
Sometimes it is easier to decide once the changes are visible. As discussed, have changed 👍